cernopendata/modules/fixtures/data/docs/cms-virtual-machine-2010/cms-virtual-machine-2010.md

```diff
@@ -18,7 +18,7 @@ Note: the latest tested version of VirtualBox working with this CMS-specific Cer
 
 **Important**: Before you download the CernVM, note that the imported settings may not always work on your host machine. Please see [Issues and Limitations](#issues2010) or [the CMS guide to troubleshooting](/docs/cms-guide-troubleshooting) if you encounter any problems with booting the VM.
 
-Next download the CMS-specific CernVM image as OVA file from: [CMS VM Image for 2010 CMS Open Data](/record/250).
+Next download the CMS-specific CernVM image as OVA file from: [CMS VM Image for 2010 CMS Open Data](/record/250). It is recommended using the version "CMS-2010-OpenData-v5".
```

Comment:
If we attach the VM image to the record:

- http://opendata.cern.ch/record/250

that it would look weird to see the image names:

- CMS-OpenData-1.0.0-rc7.ova
- CMS-2010-OpenData-v5.ova

Can we rename the new image to something resembling the old image?  For example `CMS-OpenData-1.0.1.ova` (if it fixes a minor problem) or `CMS-OpenData-1.1.0.ova` (if it is a more major update)?  This would nicely follow the usual [semantic versioning](https://semver.org/) practices...

Reply:
Good point! For the connection to the CernVM download area, it would be good if @jblomer could rename it there as well. Jakob, is that possible?

Reply:
I'll change it to CMS-OpenData-1.1.0.ova as it is a major update